javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,18 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "") {
+        console.log("Please enter a valid student name");
+    }else if (class07Students.includes(studentName)) {
+        console.log(`Student ${studentName} is already in the class`)
+    }else if(studentName === "Queen") {
```

Comment:
This condition is a pretty good candidate to be a boolean variable (`isQueen` for example). Then you can reuse it.